src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/mainscreen/MainScreenView.kt

```diff
@@ -0,0 +1,131 @@
+package org.wycliffeassociates.otter.jvm.app.ui.mainscreen
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.geometry.Orientation
+import javafx.scene.layout.*
+import org.wycliffeassociates.otter.jvm.app.images.ImageLoader
+import org.wycliffeassociates.otter.jvm.app.theme.AppStyles
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.ui.projecthome.view.ProjectHomeView
+import org.wycliffeassociates.otter.jvm.app.widgets.projectnav.projectnav
+import tornadofx.*
+
+class MainScreenView: View() {
+    override val root = hbox{}
+    var activeFragment: Workspace = Workspace()
+    var fragmentStage: AnchorPane by singleAssign()
+
+    val viewModel :MainViewViewModel by inject()
+    init {
+        importStylesheet<MainScreenStyles>()
+        activeFragment.header.removeFromParent()
+        with(root) {
+            addClass(MainScreenStyles.main)
+            style{
+                backgroundColor += AppTheme.colors.defaultBackground
+            }
+            add(projectnav{
+                style{
+                    prefWidth = 200.px
+                    minWidth = 200.px
+                }
+                navbox(messages["selectBook"], MaterialIconView(MaterialIcon.BOOK, "25px")) {
+                    var cardGraphic = ImageLoader.load(
+                            ClassLoader.getSystemResourceAsStream("images/project_image.png"),
+                            ImageLoader.Format.PNG
+                    )
+                    innercard(cardGraphic) {
+                        addClass(MainScreenStyles.navBoxInnercard)
+                        majorLabelProperty.bind(viewModel.selectedProjectName)
+                        minorLabelProperty.bind(viewModel.selectedProjectLanguage)
+                        visibleProperty().bind(viewModel.selectedProjectProperty.booleanBinding{it != null})
+                    }
+                }
+                navbox(messages["selectChapter"], MaterialIconView(MaterialIcon.CHROME_READER_MODE, "25px")) {
+                    var cardGraphic = ImageLoader.load(
+                            ClassLoader.getSystemResourceAsStream("images/chapter_image.png"),
+                            ImageLoader.Format.PNG
+                    )
+                    innercard(cardGraphic) {
+                       addClass(MainScreenStyles.navBoxInnercard)
+                        titleProperty.bind(viewModel.selectedCollectionTitle)
+                        bodyTextProperty.bind(viewModel.selectedCollectionBody)
+                        visibleProperty().bind(viewModel.selectedCollectionProperty.booleanBinding{it != null})
+                    }
+                }
+                navbox(messages["selectVerse"], MaterialIconView(MaterialIcon.BOOKMARK, "25px")) {
```

Comment:
these navboxes could be generated in a for loop and iterate over some kind of list of the varying content